### DISCUSSION TITLE
Issue with the package openxlsx

### DISCUSSION THREAD

**Discussion by RaquelMartos** [JanMarvin/openxlsx2#583] [Q&A]
I'm trying to overwrite an existing sheet in a previously created excel file (not from R). When using the following functions I get an error when I try to open the excel.

path <- "Y:\\SOURCE DATA\\TradeMonitors\\FALC\\FALC 2.0\\USFALC2018toPresent.xlsx"
wb = openxlsx::loadWorkbook(path)
openxlsx::removeWorksheet(wb, "Importers")
openxlsx::addWorksheet(wb, "Importers")
openxlsx::writeData(wb, "Importers", ImportersUSFALC)
openxlsx::saveWorkbook(wb, path, overwrite = TRUE)

When I try to open the excel file it puts me that the file is corrupted and there is part of the data that cannot be recovered. What could be the problem?

Thank you in advance.

**Comment by JanMarvin** [comment]
Sorry @RaquelMartos , but we don't have time to address `openxlsx` issues here. If you want our help please provide the XLSX input and an `openxlsx2` related question.
